ansible/roles/iptables-persistent/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
   tags: install_iptables_persistent
```

Comment:
I'm fairly sure that you'll at least have to HUP the iptables process to get it to read the new rules in.  So I don't think the notify is needless.